packages/nimble-components/src/table/testing/table.pageobject.ts

```diff
@@ -245,35 +245,18 @@ export class TablePageObject<T extends TableRecord> {
             );
         }
 
-        const collapseButton = this.getCollapseAllButton();
-        const buttonWidth = collapseButton!.getBoundingClientRect().width;
-        const buttonStyle = window.getComputedStyle(collapseButton!);
         table.style.width = `${
             rowWidth
-            + buttonWidth
-            + parseFloat(buttonStyle.marginLeft)
-            + parseFloat(buttonStyle.marginRight)
+            + (table.headerRowActionContainer.getBoundingClientRect().width
```

Comment:
Nothing on this line immediately jumps out to me as being possibly undefined. Can you double check that the `?? 0` is needed?